md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTypeAPI.java

```diff
@@ -215,9 +209,7 @@ RestWrapper insert(@ModelAttribute("processtype")
      * @return
      */
     @RequestMapping(value = {"/options/{ptid}"}, method = RequestMethod.POST)
-
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapperOptions options(@PathVariable("ptid") Integer processTypeId,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "model". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)